packages/getting-started/src/browser/getting-started-preferences.ts

```diff
@@ -0,0 +1,39 @@
+// *****************************************************************************
+// Copyright (C) 2023 Ericsson and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { interfaces } from '@theia/core/shared/inversify';
+import { nls } from '@theia/core/lib/common/nls';
+import { PreferenceSchema } from '@theia/core/lib/common/preferences/preference-schema';
+import { PreferenceContribution } from '@theia/core/lib/browser';
+
+export namespace GettingStartedPreferences {
+    export const alwaysShowWelcomePage = 'welcome.alwaysShowWelcomePage';
+}
+
+export const welcomePreferenceSchema: PreferenceSchema = {
```

Comment:
suggestion for consistency:

```suggestion
export const gettingStartedPreferenceSchema: PreferenceSchema = {
```